test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java

```diff
@@ -55,7 +57,7 @@ private void setUp(Object[] args) throws Exception {
         frame.setContentPane(desktop);
         JemmyProperties.setCurrentDispatchingModel(
                 JemmyProperties.getCurrentDispatchingModel());
-        JInternalFrame internalFrame = new JInternalFrame("JInternalFrameOperatorTest", true, true, true, true);
+        internalFrame = new UncloseableInternalFrame("JInternalFrameOperatorTest", true, true, true, true);
```

Comment:
Is it possible to make use of internalFrame.setClosable(false) API instead of overriding and having  a new class UncloseableInternalFrame.

Reply:
setClosable(boolean) only removes the close button on the window header. It does not stop window to be closed when done through the API. So, no.